src/taskpane/components/MergeComponents/MergeConflictScreen.js

```diff
@@ -0,0 +1,134 @@
+import * as React from "react";
+import { PrimaryButton } from '@fluentui/react';
+import Taskpane from "../Taskpane";
+import { headerSize } from "../../../constants";
+import MergeConflict from "./MergeConflict";
+import { runResolveMergeConflicts }  from "../../../saga/merge";
+ 
+
+
+import './MergeConflictScreen.css';
+
+/* global  */
+
+function numToChar (number) {
+    var numeric = (number - 1) % 26;
+    var letter = chr(65 + numeric);
+    var number2 = parseInt((number - 1) / 26);
+    if (number2 > 0) {
+        return numToChar(number2) + letter;
+    } else {
+        return letter;
+    }
+}
+
+function chr(codePt) {
+    if (codePt > 0xFFFF) { 
+        codePt -= 0x10000;
+        return String.fromCharCode(0xD800 + (codePt >> 10), 0xDC00 + (codePt & 0x3FF));
+    }
+    return String.fromCharCode(codePt);
+}
+
+// Take from https://stackoverflow.com/questions/9905533/convert-excel-column-alphabet-e-g-aa-to-number-e-g-25
+
+export default class MergeErrorScreen extends React.Component {
+
+  constructor(props) {
+    super(props); 
+
+    // Format Conflict Data
+    const mergeData = Object.entries(this.props.conflicts);
+    console.log(mergeData)
+    
+    let conflictsArray = []
+    mergeData.forEach((conflictData) => {
```

Comment:
I wonder, can we push this computation upstream? If we're generating merge conflict data that has to then be converted to a different format to be consumed, why don't we just generate it in a more useful format to begin with?

Reply:
Yeah, I think there is definitely some optimization we can do on formatting the data. But some formatting change is required because for individual conflicts, I want all of the data encompassed (including the sheet name), but for actually updating the sheet data, I want them grouped by sheet, so we can do all of the updates to a single sheet at once.